### PR TITLE
Add RavioliQueen

### PR DESCRIPTION
**Pull request by Spandamn**

### CODE REVIEW

File: data/mods/ssb/abilities.ts [hunk 1]
```diff
@@ -1712,6 +1712,53 @@ export const Abilities: {[k: string]: ModdedAbilityData} = {
...
+					}
+				}
+			},
+			onEnd() {
+				this.add('-fieldend', 'move: Phantom Plane');
+			},
+		},
+		isNonstandard: "Custom",
+		gen: 8,
+	},
```
Comment (KrisXV) [inline data/mods/ssb/abilities.ts]: the terrain should be its own move like electric terrain does

Comment (Spandamn) [reply data/mods/ssb/abilities.ts]: how bout make it a field condition? cause it isnt really a 'terrain', its kind of like a weather

Comment (KrisXV) [reply data/mods/ssb/abilities.ts]: it's a terrain

Comment (Spandamn) [reply data/mods/ssb/abilities.ts]: made a move

File: data/mods/ssb/abilities.ts [hunk 2]
```diff
@@ -1712,6 +1712,53 @@ export const Abilities: {[k: string]: ModdedAbilityData} = {
...
+				if (pokemon.isGrounded() && !pokemon.isSemiInvulnerable()) {
+					if (pokemon && !pokemon.hasType('Ghost')) {
+						this.damage(pokemon.baseMaxhp / 16, pokemon);
+					}
+				}
+			},
+			onEnd() {
+				this.add('-fieldend', 'move: Phantom Plane');
+			},
+		},
```
Comment (ghost) [inline data/mods/ssb/abilities.ts]:
```suggestion
```

Comment (Spandamn) [reply data/mods/ssb/abilities.ts]: oh shit lol

File: data/mods/ssb/moves.ts [hunk 3]
```diff
@@ -4107,6 +4107,90 @@ export const Moves: {[k: string]: ModdedMoveData} = {
...
+		onTryMove() {
+			this.attrLastMove('[still]');
+		},
+		onPrepareHit(target, source) {
+			this.add('-anim', source, 'Sprit Shackle', target);
+			this.add('-anim', source, 'Curse', target);
+		},
+		secondary: {
+			chance: 50,
+			volatileStatus: 'partiallytrapped',
```
Comment (ghost) [inline data/mods/ssb/moves.ts:4128]: huh, interesting.

**Pull request merged**